frontend/src/components/Breadcrumbs.tsx

```diff
@@ -59,10 +59,7 @@ const Breadcrumbs = ({ breadcrumbList }: Props) => {
   });
 
   return (
-    <GridContainer
-      className="padding-top-1 tablet:padding-top-3 desktop-lg:padding-top-4"
-      data-testid="breadcrumb"
-    >
+    <GridContainer data-testid="breadcrumb">
```

Comment:
- Breadcrumbs don't need padding classes, these are now applied in the theme settings